in_session.go

```diff
@@ -131,6 +140,14 @@ func (state inSession) handleTestRequest(session *session, msg *Message) (nextSt
 		}
 	}
 
+	if err := session.checkTargetTooLow(msg); err != nil {
+		return state
```

Comment:
dropping the `err` here? should you return `handleStateError(session, err)` instead?

Reply:
I don't think we should switch to `latentState` if we receive an out-of-sequence Test Request. We should instead handle that Test Request as we were doing till now, but just not increment the next expected target sequence number. This pattern can also be seen while handling out-of-sequence Resend Requests as can be seen [here](https://github.com/momin-javed/quickfix/blob/c855c338bde47832b273f6f6b39458f04c1d2d25/in_session.go#L226-L228).

Reply:
Just above this change in this func we already check toohigh and toolow, I think making this change redundant - https://github.com/quickfixgo/quickfix/blob/c855c338bde47832b273f6f6b39458f04c1d2d25/in_session.go#L128 let me know if I am missing something

Reply:
Ah right! Removed the change for TestRequest now:
a60af80ed30435dd26a636423ffa1cc6432ad3ca